snapshots/default.json

```diff
@@ -83,11 +86,8 @@
   "nvim-web-devicons": {
     "commit": "2d02a56"
   },
-  "tokyonight.nvim": {
-    "commit": "e0bdba5"
-  },
   "onedarker.nvim": {
-    "commit": "b00dd21"
+    "commit": "4eaa5e8"
```

Comment:
@kylo252 but we are on the `freeze` branch, shouldn't we use that commit hash?

Reply:
> @kylo252 but we are on the `freeze` branch, shouldn't we use that commit hash?

fixed!

> other than that, we can merge it whenever you like

- added a guard against enabling `document_highlight` and illuminate at the same time, just in case it's turned on for pre-existing configs.
- added a debug print just to make it clear when `document_symbols` isn't supported (to avoid complaints about it not working for some servers)